pdns/packethandler.cc

```diff
@@ -944,18 +946,21 @@ int PacketHandler::processNotify(const DNSPacket& p)
 static bool validDNSName(const DNSName &name)
 {
```

Comment:
with how much this function now assumes about how `DNSName` works, perhaps we should move it into the class at some point? Doesn't have to be in this PR, just a thought :)

Reply:
Yes, that would make a lot of sense!